meta/main.yml

```diff
@@ -26,9 +26,9 @@ dependencies:
   - role: stackhpc.os_openstacksdk
     os_openstacksdk_venv: "{{ os_images_venv }}"
     os_openstacksdk_state: "{{ os_images_package_state }}"
-    os_openstacksdk_upper_constraints: "{{ os_images_upper_constraints }}"
+    os_openstacksdk_upper_constraints_file: "{{ os_images_upper_constraints }}"
```

Comment:
Should it be os_images_upper_constraints_file?

Reply:
Good catch.